### PR TITLE
Vad probs proto

### PR DESCRIPTION
**Pull request by sarane22**
Proto changes to expose VAD probabilities to the client

### CODE REVIEW

**Review by atomer-nvidia — APPROVED** [review]
Dummy approval. Don't merge.

File: riva/proto/riva_asr.proto [hunk 1]
```diff
@@ -393,6 +393,11 @@ message StreamingRecognizeResponse {
   RequestId id = 100;
 }
 
+message PipelineStates{
```
Comment (virajkarandikar) [inline riva/proto/riva_asr.proto]:
```suggestion
message PipelineStates {
```

File: riva/proto/riva_asr.proto [hunk 2]
```diff
@@ -393,6 +393,11 @@ message StreamingRecognizeResponse {
...
+  // Neural VAD probabilities
+  repeated float vad_probabilities = 1;
```
Comment (virajkarandikar) [inline riva/proto/riva_asr.proto:398]: How does user interpret repeated vad probabilities?

**Review by atomer-nvidia — COMMENTED** [review]
LGTM. Can you give a brief on Viraj's query?

File: riva/proto/riva_asr.proto (hunk 2 above)
Comment (sarane22) [reply riva/proto/riva_asr.proto:398]: These probabilities would help the clients if they want to know when the speech/silence started. Current usecase: ACE pipeline for barge-in support.

**Review by atomer-nvidia — APPROVED** [review]

**Pull request merged**